src/components/EventBanner/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component, Fragment } from 'react';
+import { inject, observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { FormattedMessage, injectIntl, intlShape, defineMessages } from 'react-intl';
+import { Grid, Card, Typography, withStyles } from '@material-ui/core';
+import cx from 'classnames';
+import { sum } from 'lodash';
+import { Phases } from 'constants';
+
+import FavoriteButton from './FavoriteButton';
+import styles from './styles';
+import { getEndTimeCountDownString } from '../../helpers';
+
+const { BETTING, RESULT_SETTING, VOTING, FINALIZING, WITHDRAWING } = Phases;
+const messages = defineMessages({
+  pending: { id: 'str.pending', defaultMessage: 'Pending' },
+  placeBet: { id: 'bottomButtonText.placeBet', defaultMessage: 'Place Bet' },
+  setResult: { id: 'str.setResult', defaultMessage: 'Set Result' },
+  arbitrate: { id: 'bottomButtonText.arbitrate', defaultMessage: 'Arbitrate' },
+  finalizeResult: { id: 'str.finalizeResult', defaultMessage: 'Finalize Result' },
+  withdraw: { id: 'str.withdraw', defaultMessage: 'Withdraw' },
+  archived: { id: 'bottomButtonText.archived', defaultMessage: 'Archived' },
+});
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@inject('store')
+@observer
+export default class EventBanner extends Component {
```

Comment:
nit: name to `FavoriteCard`. EventBanner to me sounds like an advertisement or something.